src/ClimaLSM.jl

```diff
@@ -153,17 +156,20 @@ function make_exp_tendency(land::AbstractLandModel)
     compute_exp_tendency_list =
         map(x -> make_compute_exp_tendency(getproperty(land, x)), components)
     update_aux! = make_update_aux(land)
+    update_boundary_fluxes! = make_update_boundary_fluxes(land)
     function exp_tendency!(dY, Y, p, t)
         update_aux!(p, Y, t)
+        update_boundary_fluxes!(p, Y, t)
         for f! in compute_exp_tendency_list
             f!(dY, Y, p, t)
         end
     end
     return exp_tendency!
 end
 
+# this needs thought
```

Comment:
I think we are OK because when the user calls `initialize(model)`, `initialize_additional_lsm_aux` will get called and so those variables will appear in `p`, but with zero as value. Then in `make_set_initial_aux_state(model)`, we call and use ` update_boundary_fluxes! = make_update_boundary_fluxes(land)` , which will set the values correctly.